### ISSUE TITLE
Custom report - Chart tooltip values

### ISSUE THREAD

**Issue by jansarmir** [pimcore/pimcore#6444]
https://github.com/pimcore/pimcore/blob/986d2f25467da5b626d38a2c14fec572e09dc3b9/bundles/AdminBundle/Resources/public/js/pimcore/report/custom/report.js#L363

Tootltip for multi yAxis bar/line charts displays value of last yAxis
tooltip should be related to item.field like this
````
tooltip.setHtml(record.get(data.xAxis) + ': ' + record.get(item.field));
````

**Comment by jansarmir** [comment]
One more
**displayName** is not supported in series, **title** should be used instead

Actual **Bar** multi yAxis solution shared with **Line**, places one bar over another
So i suggest to replace it with non stacked solution, like this one
````
var series = [{
            title: titles,
            type: 'bar',
            yField: data.yAxis,
            stacked: false
        }];
````

**Comment by fashxp** [comment]
fixed via #11170
